src/storage/SplitCache/InLocalStorage.js

```diff
@@ -8,13 +8,32 @@ class SplitCacheLocalStorage {
     this.keys = keys;
   }
 
+  decrementTrafficType(split) {
+    try {
+      if (split && split.trafficTypeName) {
+        const ttKey = this.keys.buildTrafficTypeKey(split.trafficTypeName);
+        localStorage.setItem(ttKey, Number(localStorage.getItem(ttKey)) - 1);
+        if (!Number(localStorage.getItem(ttKey))) delete localStorage[ttKey];
```

Comment:
This lines could be something around:
```
const newCount = Number(localStorage.getItem(ttKey) - 1;
if (newCount > 0) localStorage.setItem(ttKey, newCount)
else localStorage.removeItem(ttKey)
```

Ideally we avoid the double operation with localstorage, also it's more correct to use the removeItem function from it instead of just cleaning the reference.